test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -106,9 +156,42 @@ var _ = g.Describe("ScyllaCluster replace", func() {
 		oldHosts := hosts
 		hosts = getScyllaHostsAndWaitForFullQuorum(ctx, f.KubeClient().CoreV1(), sc)
 		o.Expect(hosts).To(o.HaveLen(len(oldHosts)))
-		o.Expect(hosts).NotTo(o.ConsistOf(oldHosts))
 		err = di.SetClientEndpoints(hosts)
 		o.Expect(err).NotTo(o.HaveOccurred())
 		verifyCQLData(ctx, di)
-	})
+
+		framework.By("Verifying ScyllaDB config")
+
+		configClient, err := utils.GetScyllaConfigClient(ctx, f.KubeClient().CoreV1(), sc, replacedNodeService.Spec.ClusterIP)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = validateScyllaConfig(ctx, configClient, preReplaceService)
+		o.Expect(err).NotTo(o.HaveOccurred())
+	},
+		g.Entry(describeEntry, &entry{
+			procedure:             "ClusterIP",
+			scyllaImageRepository: scyllaOSImageRepository,
+			scyllaVersion:         "5.1.4",
```

Comment:
please bump to 5.1.15 if https://github.com/scylladb/scylla-operator/pull/1322 merges sooner

Reply:
#1322 has landed